rosetta-source/src/main/rosetta/base-staticdata-asset-common-type.rosetta

```diff
@@ -31,108 +31,116 @@ type Equity extends IdentifiedProduct: <"A class to specify an equity as having
 
     condition DeliveryDateParametersChoice: <"Requires definition of a delivery date or delieryNearby parameters.">
-		optional choice deliveryNearby, deliveryDate
-	
+        optional choice deliveryNearby, deliveryDate
+
 type CommodityReferenceFramework: <"Specifies the type of commodity.">
-	commodityBase string (1..1) <"Identifies the base type of the commodity being traded, using the names defined in the ISDA 2005 Commodity Definitions SubAnnex A where possible, e.g. AgriculturalProducts, CommodityCompositeIndices, Energy, Freight, Metals, or PaperAndPlastic. Implementors can define their own scheme with additional values.">
-	    [metadata scheme]
-	subCommodity string (1..1) <"Identifies the sub product type of the commodity being traded, using the names defined in the ISDA 2005 Commodity Definitions SubAnnex A where possible, e.g. under Energy, the possible values are Benzene, Coal, DieselFuel, Electricity, FuelOil, GasOil, Gasoline, HeatingOil, JetFuelKerosene, Methanol, Naphtha, NaturalGas, NaturalGasLiquids, Oil, and UltraLowSulpherDiesel. Implementors can define their own scheme with additional values.">
-	    [metadata scheme]
-	commodityName string (1..1) <"Identifies the commodity more specifically. Where possible, this should follow the naming convention used in the 2005 ISDA Commodity Definitions SubAnnex A, including the subCommodity and additional qualifiers, but should be limited to 256 characters or less.">
-	capacityUnit cdm.base.math.CapacityUnitEnum (0..1) <"Provides an enumerated value for a capacity unit, generally used in the context of defining quantities for commodities.">
-	weatherUnit cdm.base.math.WeatherUnitEnum (0..1) <"Provides an enumerated values for a weather unit, generally used in the context of defining quantities for commodities.">
-	currency string (1..1) <"Defines the currency in which the commodity is priced.">
-	    [metadata scheme]
-	
-	condition CommodityReferenceFrameworkChoice: <"Requires that either the capacity unit or weather unit is populated.">
-		optional choice capacityUnit, weatherUnit
+    commodityClassification Taxonomy (0..*) <"Identifies a commodity using one or several identification systems (ex.: EMIR Refit Table 4 of the Annex to the Comission Implementing Regulation C(2022) 3588, ISDA 2005 Commodity Definitions SubAnnex A, etc.) with any number of hierarchical layers.">
```

Comment:
Okay so commodityClassification replaces commodityBase and subCommodity, as you can now use the new ordinal to define the level of the product i.e. sub, sub sub, etc etc, got it!